projects/tabs/package.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "@skyux/tabs",
-  "version": "5.0.2",
+  "version": "5.0.3",
   "peerDependencies": {
-    "@angular/common": "^12.2.13",
```

Comment:
Being a little pedantic here. But now that whenever we upgrade these it will require our consumers to do the same. Since this change doesn't require any updates and is just a code example - I would personally advocate for just updating the version number and not bumping all of the other dependencies in this release PR.

Reply:
gotcha. closing this PR and deleting the branch so to save time undoing dependency upgrades

Reply:
closed pr to save time complying to this